server/src/configs/default.json

```diff
@@ -201,7 +201,9 @@
     "settings": {
       "userTableName": "users",
       "sessionTableName": "session",
-      "leagues": ["great", "ultra"]
+      "leagues": [],
+      "reactMapHandlesPvp": false,
+      "pvpLevels": []
```

Comment:
Why empty for default?

Reply:
So admins can choose what they want to track.

Reply:
Default should provide a reasonable default, probably not empty. 🤔